specification/personal-demographics.yaml

```diff
@@ -38,9 +38,9 @@ info:
     Unauthenticated access and citizen access are on the [roadmap](https://nhs-digital-api-management.featureupvote.com/?order=popular&filter=allexceptdone&tag=pds-fhir-api).
 
     ## Legal use
-    This API can only be used where there is a [legal basis](http://digital.nhs.uk/services/demographics/access-data-on-the-personal-demographics-service) to do so. You should [make a PDS access request](https://digital.nhs.uk/services/demographics/access-data-on-the-personal-demographics-service) before you go too far with your development.
+    This API can only be used where there is a [legal basis](http://digital.nhs.uk/services/demographics/access-data-on-the-personal-demographics-service) to do so. Make sure you have a valid use case before you go too far with your development.  To do this, make a [PDS access request](https://digital.nhs.uk/services/demographics/access-data-on-the-personal-demographics-service). 
     
-    You must have made this request before you can go live (see ‘Onboarding’ below).
+    You must do this before you can go live (see ‘Onboarding’ below)
```

Comment:
missing a full stop?
we have them in our other paragraphs? Or do you have a grammar fact you're going to hit me with here @mick-sch ?  ;-)

Reply:
Good spot!